src/NzbDrone.Core/Indexers/Newznab/NewznabRequestGenerator.cs

```diff
@@ -404,17 +404,45 @@ public virtual IndexerPageableRequestChain GetSearchRequests(AnimeEpisodeSearchC
                         "search",
                         $"&q={NewsnabifyTitle(queryTitle)}+{searchCriteria.AbsoluteEpisodeNumber:00}"));
 
-                    if (Settings.AnimeStandardFormatSearch && searchCriteria.SeasonNumber > 0 && searchCriteria.EpisodeNumber > 0)
+                    if (Settings.AnimeStandardFormatSearch && searchCriteria.SeasonNumber > 0)
```

Comment:
A separate search is required, right now this will get called for every episode in the season.

Reply:
I made it so that it does another search separately in `ReleaseSearchService` for the season alone.